packages/client/utils/draftjs/getTypeFromEntityMap.ts

```diff
@@ -13,8 +13,8 @@ const getTypeFromEntityMap = (
   const typeSet = new Set<string>()
   const id = keyLookup[type]
   for (let i = 0; i < entityKeys.length; i++) {
```

Comment:
nit: `Object.entries`

Reply:
As I see, `key` is not needed here, so will use `Object.values(entityMap).forEach((entity)`